tests/ctst/steps/notifications.ts

```diff
@@ -49,41 +76,51 @@ async function emptyVersionedBucket(world: IWorld) {
     }));
 }
 
-async function putObject(world: IWorld) {
+async function putObject(world: Zenko) {
```

Comment:
Mixed with [the logic we currently have](https://github.com/scality/Zenko/pull/1820/files#diff-504fd6b6f773cd1c920c1a4789386463c448655cdeb9ca7395ae4108cd787d6aR8), the World may not always be Zenko...